_posts/2015-04-03-april-newsletter.md

```diff
@@ -0,0 +1,285 @@
+---
+layout: newsletter
+title: "Newsletter #5 - TODO"
+category: newsletter
+permalink: /news/2015/april/
+---
+
+Welcome to the fifth newsletter for Neovim.
+
+## General News
+
+### Black Duck 2014 Open Source Rookie
+
+[Black Duck][blackduck-site], a company that helps other corporations in the
+adoption of open source software, chose Neovim as one of the projects to feature
+of this last year. The company annually recognizes some of the top new open
+source projects that started during the year.
+
+### ICCF Holland Donations
+
+Mentioned in the last letter, [Unixstickers][iccf-unixstickers] has added
+[Neovim stickers][iccf-stickers] to their inventory. Every sticker that is
+bought increases the amount donated to [ICCF Holland][iccf-site], Bram
+Moolenaar's charity.
+
+Since they've been added, USD 106 has been raised which Unixstickers [just
+donated][iccf-tweet] on March 24th.
+
+### Related Projects
+
+The [Neovim Wiki][related-main-wiki] contains a page titled [Related
+projects][related-wiki] that contains a list of current GUI projects, API
+clients, plugins that take advantage of Neovim features, and projects with
+direct support for Neovim.
+
+Some of the projects might be experimental which means there might be bugs and
+the projects might still be in progress.
+
+Do you know of another project that can be listed? Don't be afraid to add a link
+yourself!
+
+## Development News
+
+### Replacing Terminal UI
+
+@tarruda has merged his [pull request][tui-pr] to reimplement the terminal user
+interface using [libtermkey][tui-libtermkey] and [unibilium][tui-unibilium].
+
+Libtermkey is a library by @leonerd that "allows easy processing of keyboard
+entry from terminal-based programs." It has the ability to handle special keys,
+[UTF-8 combining][tui-combining] and more.
+
+Unibilium is a library that interfaces with [terminfo][tui-terminfo]. Terminfo
+is a library and database that makes it easy to interface with display terminals
+without worrying about device specific details. More details can be found on the
+man page for terminfo by running: [`man 5 terminfo`][tui-manpage].
+
+#### Additional Keybindings
+
+As a result of using libtermkey, Neovim can now bind [meta keys][tui-meta] to
```

Comment:
I think this makes it sounds like it's primarily meta keys, when it's conceptually more than just that IMO.